Orchestration/OrchestrationService/ModuleConfigurationDeployment.ps1

```diff
@@ -13,6 +13,9 @@
     [string]
     $WorkingDirectory,
     [Parameter(Mandatory=$false)]
+    [string]
+    $ToolkitConfigurationFilePath = './Config/toolkit.config.json',
```

Comment:
This is still in linux format, make sure to do:

1. Split based on "/"
2. Use the array to call Join-Path to create an OS agnostic relative path

Reply:
Made changes as per our discussion.